### PR TITLE
Replace jtssql with jsontableschema

### PR DESCRIPTION
**Pull request by akariv**
@roll

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
         'sqlalchemy',
         'click',
         'datapackage',
-        'jtssql',
+        'jsontableschema',
```
Comment (roll) [inline setup.py]: PR's code only need `jsontableschema-sql` package but it's not published on PyPi for now (we've just discussed it on FD slack channel=)

Comment (akariv) [inline setup.py]: I think we'll also have to wait for all constraints to be implemented as well.

Just wanted to make sure I'm using the API correctly.

**Comment by roll** [comment]
API usage looks correct

**Comment by akariv** [comment]
Thanks :)

**Pull request merged**